app/src/main/res/values/strings.xml

```diff
@@ -164,13 +164,13 @@
     <string name="instance_summary">Piped, login, subscriptions</string>
     <string name="customInstance_summary">Add a custom instance (on your own risk)</string>
     <string name="instance_name">Instance name</string>
-    <string name="instance_api_url">Instance API url</string>
+    <string name="instance_api_url">Instance API URL</string>
     <string name="addInstance">Add Instance</string>
-    <string name="empty_instance">You have to fill in the name and the API url.</string>
+    <string name="empty_instance">Fill in the name and the API URL.</string>
     <string name="clear_customInstances">Clear custom instances</string>
-    <string name="invalid_url">Please enter a valid url</string>
+    <string name="invalid_url">Please enter a URL that works</string>
     <string name="version">Version %1$s</string>
     <string name="about_summary">Get to know team LibreTube and how it all happens.</string>
     <string name="related_streams">Related streams</string>
-    <string name="related_streams_summary">Show related streams to videos.</string>
+    <string name="related_streams_summary">Show streams related to videos you watch.</string>
```

Comment:
```suggestion
    <string name="related_streams_summary">Show content related to what you watch.</string>
```

Reply:
This could be changed to "the video you are watching currently", or something similar.